workflow/common.go

```diff
@@ -0,0 +1,72 @@
+package workflow
+
+import (
+	"github.com/hpe-storage/nimble-golang-sdk/pkg/client/v1/nimbleos"
+	"github.com/hpe-storage/nimble-golang-sdk/pkg/param"
+	"github.com/hpe-storage/nimble-golang-sdk/pkg/service"
+)
+
+var defaultVolumeName = "DefaultVolumeTest"
+var defaultInitiatorGrpName = "DefaultInitiatorgrpTest"
+var defaultVolCollName = "DefaultVolCollTest"
+
+func config() (*service.NsGroupService, error) {
+	groupService, err := service.NewNsGroupService("", "admin", "admin", "v1", true)
+	if err != nil {
+		return groupService, err
+	}
+	defer groupService.LogoutService()
```

Comment:
This causes a logout to be performed at the end of this function.  I don't think that is wanted.